modules/mono/glue/GodotSharp/GodotSharp/Core/Dictionary.cs

```diff
@@ -352,7 +352,7 @@ public override string ToString()
     /// </summary>
     /// <typeparam name="TKey">The type of the dictionary's keys.</typeparam>
     /// <typeparam name="TValue">The type of the dictionary's values.</typeparam>
-    public class Dictionary<TKey, TValue> :
+    public class Dictionary<[Godot.MustBeVariant] TKey, [Godot.MustBeVariant] TValue> :
```

Comment:
Is the namespace necessary?

Reply:
I think there might've been a conflict with something if I added the `using Godot` but I'll check.

Reply:
I don't think so.